.github/workflows/pr-percy-prepare-push.yml

```diff
@@ -0,0 +1,25 @@
+name: "Prepare Percy build on Percy - push"
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - "templates/docs/examples/**"
+      - "scss/**"
+    types:
+      - opened
+      - ready_for_review
+      - synchronize
+
+jobs:
+  copy_artifact:
+    name: Copy changed files to GHA artifact
+    # Ignore draft PRS and PRs with the Percy label
+    # If we run tests against PRs with the Percy label, we will run tests twice (test is also ran by the labelling workflow)
+    if: ${{ !github.event.pull_request.draft && !contains(github.event.pull_request.labels.*.name, vars.PERCY_TEST_REQUESTED_LABEL_NAME) }}
```

Comment:
Would it be possible to have one workflow and construct the conditional in a way that would cover the cases we need?

Right now it seems they are both very similar, and the only difference is in existence of the label? But I guess I'm missing something?

Reply:
@bartaz 
The reason two are needed is that the `push` workflow is set to only run if the PR changes relevant Percy files, while the label workflow runs on every push to a PR that has the Percy label, regardless of which files were changed. If I have a "paths" selector, on a workflow that needs to run against PRs with the Percy label, the workflow won't run, since the "paths" selector takes precedence.